Firmware/Marlin_main.cpp

```diff
@@ -3800,6 +3800,26 @@ static void gcode_G92()
     }
 }
 
+/// @brief Helper function to reduce code size in M861 
+/// by extracting common code into one function
+static void gcode_M861_print_pinda_cal_eeprom() {
+    int16_t usteps;
+    for (uint8_t i = 0; i < 6; i++) {
+        usteps = 0;
+        if(i > 0) {
+            usteps = eeprom_read_word((uint16_t*) EEPROM_PROBE_TEMP_SHIFT + (i - 1));
+        }
+        float mm = ((float)usteps) / cs.axis_steps_per_unit[Z_AXIS];
+        i == 0 ? SERIAL_PROTOCOLPGM("n/a") : SERIAL_PROTOCOL(i - 1);
+        SERIAL_PROTOCOLPGM(", ");
```

Comment:
Please check, that gcc is able to merge the duplicate `", "` strings.

Reply:
@DRracer do you mean by adding `", "` to `messages.cpp`?

Reply:
Even by extracting to a `static const char something[] PROGMEM;` locally ...

Reply:
... and yes, having a collection of these tiny strings at one spot and reusing them across the whole firmware might actually bring some savings.

Reply:
Done ✔️